knowledge-base/drawer-template-as-side-navigation.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Could be linked from the Template article of the drawer too

Reply:
It is already linked.